### DISCUSSION TITLE
Will picture_output best work with "pause on" and emulate_motion?

### DISCUSSION THREAD

**Discussion by tukey42** [Motion-Project/motion#1738] [Q&A]
I am trying to use the built in motion detection of an ip camera together with "motion". Therefore "pause" is set to on and a script will trigger "emulate_motion=on".  I have the feeling, that in this case "picture_output best" will not work. Instead the first image is used. Is this true?

Many thanks for your help!

**Comment by tosiara** [comment]
I never tried. Can you try a let everyone know if that works as expected?

**Reply by tukey42** [reply]
I checked the videos and taken pictures of my cameras. If I am not wrong, the pictures taken by "motion" from the cameras where "motion" is detecting the motion are the ones with the most changes, while the pictures taken by "motion" from the cameras where the motion is detected by the cameras and triggering "motion" with "emulate_motion=on" are always a few frames from the beginning of the video.

**Reply by tosiara** [reply]
I have checked this with "pause" and "start" commands, and it correctly saves the best shot. If you have issues, please provide your config and full log to verify

**Reply by tosiara** [reply]
Ah, now I see your point. So you change emulate_motion configuration parameter to initiate an event? Yes, in that case I can see that the best shot will be always the first one, because when motion emulate is on - it is not doing any evaluation of motion area, all pixels are treated as changed, and the first biggest amount of changed pixels is marked as best.
